pkg/common/vsphere/virtual_center_test.go

```diff
@@ -0,0 +1,222 @@
+package vsphere
+
+import (
+	"context"
+	"github.com/sirupsen/logrus"
+	"github.com/vmware-tanzu/astrolabe/pkg/util"
+	"github.com/vmware/govmomi/session"
+	"k8s.io/client-go/tools/clientcmd"
+	"os"
+	"testing"
+	"time"
+)
+
+/*
+	Steps
```

Comment:
Would you please comment how to run these testcases somewhere?

Reply:
done.